conan/tools/cmake/toolchain/blocks.py

```diff
@@ -313,7 +313,7 @@ def context(self):
         android_ndk_path = android_ndk_path.replace("\\", "/")
 
         ctxt_toolchain = {
-            'android_platform': self._conanfile.settings.os.api_level,
+            'android_platform': 'android-' + str(self._conanfile.settings.os.api_level),
```

Comment:
Android is not sufficiently tested in our CI. 
Are we completely sure that this format ``android-xxx`` does not break for more modern API levels? It is compatible for all current (and hopefully future) levels?

Reply:
As far as it's shown in the official documentation, https://developer.android.com/ndk/guides/cmake?hl=es-419#android_platform is not saying anything about "restrictions" in terms of using `android-xxx` for modern versions, so it should work as usual.

From official docs:
```
Multiple formats are accepted for this parameter:

* android-$API_LEVEL
* $API_LEVEL
* android-$API_LETTER

The $API_LETTER format allows you to specify android-N without the need to determine the number associated with that release. Note that some releases received an API increase without a letter increase. These APIs can be specified by appending the -MR1 suffix. For example, API level 25 is android-N-MR1.
```